### PR TITLE
Merge develop 6.7 07.12.15

### PR DESCRIPTION
**Pull request by livecodeali**
Conflicts:
docs/dictionary/operator/is-a.xml
engine/src/parseerrors.h

### CODE REVIEW

**Comment by livecodeali** [comment]
@livecode-vulcan review by @peter-b ok acf7fca

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok acf7fca8da48a0ad5ebe23073fa70837fca8c541

File: engine/src/parseerrors.h [hunk 1]
```diff
@@ -1706,6 +1706,11 @@ enum Parse_errors
 	// MDW-2014-08-23: [[ feature_floor ]] ceil: error in source expression
 	// {PE-0554} ceil: error in source expression
 	PE_CEIL_BADPARAM,
+
+    // SN-2015-11-15: [[ Bug 165452 ]] New error, if a global variable shadows
+    // a local variable declared beforehand
+    // {PE-0545} global: shadowing a local variable
```
Comment (livecodesebastien) [inline engine/src/parseerrors.h]: (Just looking by curiosity) I think that the index should be updated to 555

**Comment by livecode-vulcan** [comment]
😎 test success acf7fca8da48a0ad5ebe23073fa70837fca8c541
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1390)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1410)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1386)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1396)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1407)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1406)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1411)

**Comment by livecodeali** [comment]
@livecodesebastien Thanks!
@livecode-vulcan review by @peter-b ok d968b02

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok d968b025e9b06c737ad6cfd114d2e5bb9bc8206e

**Comment by livecode-vulcan** [comment]
😎 test success d968b025e9b06c737ad6cfd114d2e5bb9bc8206e
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1391)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1411)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1387)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1397)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1408)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1407)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1412)

**Pull request merged**